src/interpreter/ByteCodeGenerator.cpp

```diff
@@ -151,7 +173,8 @@ ByteCodeBlock* ByteCodeGenerator::generateByteCode(Context* c, InterpretedCodeBl
 
 #ifdef ESCARGOT_DEBUGGER
     ByteCodeBreakpointContext breakpointContext;
-    breakpointContext.m_lastBreakpointLine = 0;
+    breakpointContext.m_lastBreakpointLine = codeBlock->asInterpretedCodeBlock()->functionStart().line;
+    breakpointContext.m_lastBreakpointIndex = codeBlock->asInterpretedCodeBlock()->functionStart().index;
```

Comment:
I'm wondering that initialization of each location info (`m_lastBreakpointLine`, `m_lastBreakpointIndex`) just to zero would have any issue. If this approach is available, then default constructor of `ByteCodeBreakpointContext` looks more proper at here.